src/book/02-system/02-subsystems/04-motion.mdx

```diff
@@ -63,7 +116,7 @@ emit(std::make_unique<ExecuteScript>(id, script, NUClear::clock::now()));
 
 The [ScriptEngine](https://github.com/NUbots/NUbots/tree/96357855b059cd499021552b3e25a4b158828dc5/module/motion/ScriptEngine) module receives either of these messages and processes the information for the script. It collects the servo positions and time for each frame and modifies them if needed based on the given duration and start time in the message. The information is then emitted as a [ServoCommand](https://github.com/NUbots/NUbots/blob/cbb5d3ebc2a7c56944ec9afe3f70faf7e2900217/shared/message/behaviour/ServoCommand.proto) message.
 
-## Script files
+### Script files
```

Comment:
Field descriptions could be more descriptive
 - duration: Length of time it will take this frame to complete. An integer in milliseconds.
 - position: The target position of the servo. A float in radians.
 - gain: Amount of effort the servo will use to get to the target position. Corresponds to the P gain of the servo. Gain is often between 10 and 30.